run_pacta_data_preparation.R

```diff
@@ -204,7 +204,8 @@ if (config[["update_currencies"]]) {
     currencies_preflight_data_path = currencies_preflight_data_path
   )
   currencies <- pacta.data.scraping::get_currency_exchange_rates(
-    quarter = config[["imf_quarter_timestamp"]]
+    quarter = config[["imf_quarter_timestamp"]],
+    max_seconds = 600L
```

Comment:
is 10 minutes a reasonable amount of waiting time to bake in to the code? honestly, not sure what makes sense / what you're comfortable with. I'm ok with it if you're ok with it

Reply:
I mean, most of the time it should be ~0. I don't think that under normal conditions this will have a big effect.